### PR TITLE
Run rustfmt on everything

### PR DESCRIPTION
**Pull request by mandreyel**
I was reading through the code and noticed that the formatting was different from what I understand to be the currently agreed upon convention. So I ran rustfmt on the entire repo. I hope that's welcome, it made reading the code more pleasant. Tests run as previously.

### CODE REVIEW

File: html5ever/examples/arena.rs [hunk 1]
```diff
@@ -147,19 +152,26 @@ impl<'arena> Sink<'arena> {
...
     {
         let new_node = match child {
             NodeOrText::AppendText(text) => {
                 // Append to an existing Text node if we have one.
-                if let Some(&Node { data: NodeData::Text { ref contents }, .. }) = previous() {
+                if let Some(&Node {
+                    data: NodeData::Text { ref contents },
+                    ..
+                }) = previous()
+                {
```
Comment (mandreyel) [inline html5ever/examples/arena.rs]: I'm not happy about the placement of some of the newlines, like the one here. Maybe I could manually adjust these cases.

File: html5ever/examples/noop-tree-builder.rs [hunk 2]
```diff
@@ -35,14 +36,17 @@ impl Sink {
...
+        self
+    }
 
     fn get_document(&mut self) -> usize {
         0
     }
 
     fn get_template_contents(&mut self, target: &usize) -> usize {
-        if let Some(expanded_name!(html "template")) = self.names.get(&target).map(|n| n.expanded()) {
+        if let Some(expanded_name!(html "template")) = self.names.get(&target).map(|n| n.expanded())
```
Comment (mandreyel) [inline html5ever/examples/noop-tree-builder.rs]: The call chain could probably broken up into separate lines.

**Comment by nox** [comment]
I don't think we should merge this PR if we don't enforce the same rules as servo/servo with the same settings.

**Comment by mandreyel** [comment]
@nox Oh, I missed that Servo has its own rusfmt config. I added it to this project and rerun the formatter. What do you think?

**Comment by Ygg01** [comment]
@mandreyel It seems some updates broke this PR, would you like to retry this?

**Comment by mandreyel** [comment]
@Ygg01 Yes, sorry, I was a bit busy and forgot about this. I'll get to it tonight or tomorrow.

**Comment by Ygg01** [comment]
Oh, no problem, I'm sorry you waited as long as you did.

**Comment by mandreyel** [comment]
Oh, damn, forgot to run tidy again...

**Comment by mandreyel** [comment]
Alright, done :)

**Pull request merged**